OracleSOASuite/dockerfiles/12.2.1.4/README.md

```diff
@@ -150,256 +142,252 @@ The Oracle database server container requires custom configuration parameters fo
 To run the database container to host the RCU schemas:
 
 1.  Add the following parameters to a `db.env.txt` file:
-
-        DB_SID=soadb
-
-        DB_PDB=soapdb
-
-        DB_DOMAIN=us.oracle.com
-
-        DB_BUNDLE=basic
-
+    ``` bash
+    DB_SID=soadb
+    DB_PDB=soapdb
+    DB_DOMAIN=us.oracle.com
```

Comment:
Please do not use real FDQDNs. This should be changed to `example.com`

Reply:
Corrected in latest commit. Kindly review.